src/backend/TrafficCourts/Staff.Service/Controllers/ComsController.cs

```diff
@@ -162,4 +162,62 @@ public async Task<IActionResult> UploadDocumentAsync([FromForm] FileUploadReques
             return new HttpError(StatusCodes.Status500InternalServerError, e.Message);
         }
     }
+
+    /// <summary>
+    /// Downloads a document for the given unique file ID if the virus scan staus is clean.
+    /// </summary>
+    /// <param name="fileId">Unique identifier for a specific document.</param>
+    /// <param name="cancellationToken"></param>
+    /// <response code="200">The document is successfully downloaded.</response>
+    /// <response code="400">The request was not well formed. Check the parameters.</response>
+    /// <response code="401">Unauthenticated.</response>
+    /// <response code="403">Forbidden, requires jjdispute:read permission.</response>
+    /// <response code="500">There was a server error that prevented the file to be downloaded successfully.</response>
+    /// <returns>The document</returns>
+    [HttpGet("DownloadDocument")]
```

Comment:
Are endpoint URLs case sensitive? I think all the endpoints (as reported by Swagger) are lowercase.  Should we be consistent?

Reply:
The endpoint URLs are case insensitive so it will appear as all lowercase in the Swagger UI. This is just for better readability in the code.